src/lang/checkers/Cashflow.ml

```diff
@@ -598,6 +598,39 @@ module ScillaCashflowChecker
                 (* Error *)
                 [[ ( Inconsistent , List.map (fun _ -> Inconsistent) args_tags ) ]] in
           (c_r_sigs, c_as_sigs)
+      | "pow" ->
+          let c_r_sigs =
+            match res_tag with
+            | NotMoney -> [ ( NotMoney     , [ NotMoney ; NotMoney ] ) ]
+            | Money    -> [ ( Money        , [ Money    ; NotMoney ] ) ]
```

Comment:
I'm not sure about this part of the signature. For multiplication we don't use that `Money` times `Money` is `Money`, so multiplying more `Money` on top of it shouldn't make it more like `Money`.

I think `pow` should only ever have the signature `(NotMoney, [ NotMoney; NotMoney])`, with appropriate adjustments to the other cases in the pattern matches.